capstone-app/src/app/modules/photos/components/photos-section/photos-section.component.ts

```diff
@@ -34,26 +36,26 @@ export class PhotosSectionComponent implements OnInit, OnChanges {
   navItems: NavItem[] = [];
   markerFilter = '';
 
+  constructor(private readonly cd: ChangeDetectorRef, private places: SharedPlacesCityService) { }
```

Comment:
private readonly cd: ChangeDetectorRef

Is this being used anywhere in this component? If not, let's remove